.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+**Summary:**
+    Short summary of key additions or changes or fixes, including public facing issue
+    or bug being address if it exists
+
+**Details:**
+
+ - list of key changes to aide present and future reviewers in understanding what
+ - is happening in this PR
+
+**Merge Checklist:**
+
+ - [ ] Passing CI
+ - [ ] Update documentation or README.md
+ - [ ] Additional Test/example added (if applicable) and passing
+ - [ ] At least one reviewer approval
+ - [ ] (optional) Clang sanitizer scan run and triaged
+ - [ ] Clang formatter applied
```

Comment:
formatter might not be needed here because the current CI checks that.

Reply:
yeah, maybe I point out it is part of CI passing, but i wanted to mention it because running the formatter is on the PR creator,  CI only verifies it has been run ...

Reply:
```suggestion
 - [ ] Clang formatter applied (verified as part of passing CI)
```